<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flocking System: Joshua Bufton</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="Controls">
      <property name="title">
       <string>Other Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of Drops to create per frame</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="DropsPerTick">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="BoidCreation">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QPushButton" name="DeletePlane">
         <property name="text">
          <string>Delete Plane</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Wind Y Direction</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="NewPlaneY">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="WindX">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Wind X Direction</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QDoubleSpinBox" name="BoxSizeX">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QDoubleSpinBox" name="BoxSizeY">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="WindY">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Wind Z Direction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="CreatePlane">
         <property name="text">
          <string>Create Plane</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="WindZ">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.000010000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="NewPlaneXSize">
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>NewPlaneZ</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="NewPlaneZSize">
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>NewPlaneZRot</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>NewPlaneRoughness</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>NewPlaneXSize</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="NewPlaneXRot"/>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>NewPlaneZSize</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>NewPlaneXRot</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="NewPlaneRoughness"/>
       </item>
       <item row="8" column="2">
        <widget class="QDoubleSpinBox" name="NewPlaneZRot"/>
       </item>
       <item row="12" column="3">
        <widget class="QDoubleSpinBox" name="BoxSizeZ">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="NewPlaneX">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="NewPlaneZ">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>NewPlaneY</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>NewPlaneX</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>BoxSizeX</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>BoxSizeY</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>BoxSizeZ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
